syntax = "proto3";

package pubsub;

// Processed data, ready to be served to the model.
message Processed {
  repeated float coefficients = 1;
  uint32 metadata_version = 100;
  uint32 timestamp = 101;
  uint32 segment_id = 102;
}